config/routes.rb

```diff
@@ -2,6 +2,7 @@
   devise_for :users, controllers: { registrations: 'users/registrations', confirmations: 'users/confirmations' }
   get 'home/index'
   root 'home#index'
+  get 'users/homepage'
```

Comment:
This works but it's better to route using resources as this is from the `UsersController`.
```
resources :users, only: [] do
    collection do
      get :homepage
    end
  end
```